modules/gltf/src/lib/extensions/EXT_structural_metadata.ts

```diff
@@ -298,55 +318,31 @@ function processPropertyTable(
 
 /**
  * Decodes properties from binary sourse based on property type.
```

Comment:
- A bit more context? Properties for what? 
- Binary source => binary data?
- source is misspelled
```suggestion
 * Decodes properties from binary source based on property type.
```

Reply:
Done